fast64_internal/f3d/f3d_gbi.py

```diff
@@ -2580,9 +2594,9 @@ def to_c_vertex_scroll(self, gfxFormatter: GfxFormatter) -> CScrollData:
 
         return data
 
-    def to_c_gfx_scroll(self, gfxFormatter: GfxFormatter) -> CScrollData:
+    def to_c_gfx_scroll(self, gfxFormatter: GfxFormatter, inline: bool = False) -> CScrollData:
         data = CScrollData()
-        if not self.inline:
+        if not inline:
             for _, (fMaterial, _) in self.materials.items():
                 fMaterial: FMaterial
                 data.append(gfxFormatter.gfxScrollToC(fMaterial.material, self.f3d))
```

Comment:
Both `fMaterial.material` and `fMesh.draw` can be handled (works for either bleed on/off) as long as `fMaterial.material` is None checked, letting you remove `inline` from this.

Reply:
good call